open_policy_agent/README.md

```diff
@@ -0,0 +1,151 @@
+
+### Installation
+
+To install the open_policy_agent check on your Kubernetes cluster:
+
+1. Install the [developer toolkit][3].
+2. Clone the `integrations-extras` repository:
+
+   ```shell
+   git clone https://github.com/DataDog/integrations-extras.git.
+   ```
+
+3. Update your `ddev` config with the `integrations-extras/` path:
+
+   ```shell
+   ddev config set extras ./integrations-extras
+   ```
+
+4. To build the `open_policy_agent` package, run:
+
+   ```shell
+   ddev -e release build open_policy_agent
+   ```
+
+5. [Download the Agent manifest to install the Datadog Agent as a DaemonSet][4].
+6. Create two `PersistentVolumeClaim`s, one for the checks code, and one for the configuration.
+7. Add them as volumes to your Agent pod template and use them for your checks and configuration:
+
+   ```yaml
+        env:
+          - name: DD_CONFD_PATH
+            value: "/confd"
+          - name: DD_ADDITIONAL_CHECKSD
+            value: "/checksd"
+      [...]
+        volumeMounts:
+          - name: agent-code-storage
+            mountPath: /checksd
+          - name: agent-conf-storage
+            mountPath: /confd
+      [...]
+      volumes:
+        - name: agent-code-storage
+          persistentVolumeClaim:
+            claimName: agent-code-claim
+        - name: agent-conf-storage
+          persistentVolumeClaim:
+            claimName: agent-conf-claim
+   ```
+
+8. Deploy the Datadog Agent in your Kubernetes cluster:
+
+   ```shell
+   kubectl apply -f agent.yaml
+   ```
+
+9. Copy the integration artifact .whl file to your Kubernetes nodes or upload it to a public URL
+
+10. Run the following command to install the integrations wheel with the Agent:
+
+    ```shell
+    kubectl exec ds/datadog -- agent integration install -w <PATH_OF_OPEN_POLICY_AGENT_ARTIFACT_>/<OPEN_POLICY_AGENT_ARTIFACT_NAME>.whl
+    ```
+
+11. Run the following commands to copy the checks and configuration to the corresponding PVCs:
+
+    ```shell
+    kubectl exec ds/datadog -- sh
+    # cp -R /opt/datadog-agent/embedded/lib/python2.7/site-packages/datadog_checks/* /checksd
+    # cp -R /etc/datadog-agent/conf.d/* /confd
+    ```
+
+12. Restart the Datadog Agent pods.
+
+### Logs-generated metrics
+
+The default dashboard includes some graphs related to a metric around OPA decisions, called `open_policy_agent.decisions`. This metric is created based on the OPA "Decision Logs". To generate this metric and populate this part of the dashboard, you will need to create a new log-generated metric in Datadog.
+
+First, we will create a facet for the `msg` field of our OPA logs, as we will only generate metrics for the "Decision Logs" type of log entry. For that, select any of the log entries coming from OPA, click on the engine log near the `msg` field and select "Create facet for @msg":
```

Comment:
```suggestion
First, create a facet for the `msg` field of our OPA logs, as it will only generate metrics for the "Decision Logs" type of log entry. For that, select any of the log entries coming from OPA, click on the engine log near the `msg` field and select "Create facet for @msg":
```